plugins/icons_suruplus_aspromauros/change_color.sh

```diff
@@ -136,21 +139,27 @@ fi
 
 if [ -n "${ICONS_SYMBOLIC_PANEL:-}" ]; then
 	echo ":: Replacing symbolic panel colors..."
-	find "$tmp_dir"/Suru++-Asprómauros/panel/{16,22,24} \
-		"$tmp_dir"/Suru++-Asprómauros/animations/{22,24} \
+	find "$tmp_dir"/Suru++-Asprómauros/animations/{22,24} \
 		-type f -name '*.svg' \
 		-exec sed -i'' -e "s/d3dae3/$ICONS_SYMBOLIC_PANEL/g" '{}' \;
 fi
 
 if [ "$SURUPLUS_GRADIENT_ENABLED" = "true" ] && [ -n "${SURUPLUS_GRADIENT1:-}" ] && [ -n "${SURUPLUS_GRADIENT2:-}" ]; then
 	echo ":: Replacing gradient colors..."
-	find "$tmp_dir"/Suru++-Asprómauros/apps/16 \
-		"$tmp_dir"/Suru++-Asprómauros/devices/16 \
+	find "$tmp_dir"/Suru++-Asprómauros/actions/{16,22,24,symbolic} \
+		"$tmp_dir"/Suru++-Asprómauros/apps/{16,symbolic} \
+		"$tmp_dir"/Suru++-Asprómauros/devices/{16,symbolic} \
+		"$tmp_dir"/Suru++-Asprómauros/emblems/symbolic \
+		"$tmp_dir"/Suru++-Asprómauros/emotes/symbolic \
 		"$tmp_dir"/Suru++-Asprómauros/mimetypes/16 \
-		"$tmp_dir"/Suru++-Asprómauros/places/16 \
+		"$tmp_dir"/Suru++-Asprómauros/panel/{16,22,24} \
+		"$tmp_dir"/Suru++-Asprómauros/places/{16,symbolic} \
+		"$tmp_dir"/Suru++-Asprómauros/status/symbolic \
 		-type f -name '*.svg' \
 		-exec sed -i'' \
-			-e 's/currentColor/url(#oomox)/g' \
+		# For avoiding to overwrite `fill|stroke="currentColor" class="ColorScheme-Highlight"`
```

Comment:
you can't put comments between `\` escaped splitlnes

Reply:
Done